### DISCUSSION TITLE
Better authentication system that lets regular users can see the page, and admins can log in and skip songs.

### DISCUSSION THREAD

**Discussion by IoIxD** [jcorporation/myMPD#717] [Ideas]
I do quite like how the interface looks, and it would be nice to direct users to this interface instead of icecast2's. However I obviously don't want everyone to be able to skip the song, change settings, etc. So It would be nice if there was support for a proper admin system, instead of relying on apache2's rudimentary auth.

**Comment by jcorporation** [comment]
You can set a pin to protect settings, scripts, etc.

https://jcorporation.github.io/myMPD/configuration/pin-protection
